CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## v.1.4.1
+
+### 18/06/2020
+
+### Fix
```

Comment:
Not exactly a fix to me

Reply:
In fact, the new version of the Core SDK fixes an issue in the widget. Would you prefer that I specify the bug in the changelog?